wrensec-guice-core/src/test/java/org/forgerock/guice/core/test/TestModule1.java

```diff
@@ -22,7 +22,6 @@
 public class TestModule1 implements Module {
 
     private TestModule1() {
-
```

Comment:
This seems unnecessary. Why hasn't a change of this kind been made in other modules too?